addons/fabimakesgames.embed_game/embed_game.gd

```diff
@@ -246,10 +246,16 @@ func set_window_mode(mode: DisplayServer.WindowMode):
 func get_top_buttons() -> Array[Node]:
 	var cont := Control.new()
 	add_control_to_container(CustomControlContainer.CONTAINER_TOOLBAR, cont)
-	var btns := cont.get_parent().get_child(2).get_children()
+	var btns := cont.get_parent().get_child(get_top_buttons_position()).get_children()
 	remove_control_from_container(CustomControlContainer.CONTAINER_TOOLBAR, cont)
 	return btns
 	
+func get_top_buttons_position() -> int:
+	var top_buttons_position = 2
```

Comment:
I wonder if there is a more robust way to do this, for example if mac or any OS changes their button layout this will fail.
Is there a way to identify the container? If there is, we can do a loop and search for that specific container.

Reply:
Totally agree ( even for linux user that can have the mac buttons or the windows buttons even in the middle for certain distrib ) 

I made this as a quickfix to allow macOs contributors/users to quickly come into the plugin, in fact we can identify the tops buttons by the name of the HBox used, but same, if godot change the name of this hbox it will also fail, both have potential fail situation ( but I think that mac OS test was a "longer" fix than Godot naming, since Godot make lot of work in depth for such thing than Apple Developer on macOs )

maybe there is another solution but I didn't explore more for now

Reply:
Makes me wonder if ultimately a different way  of adding the button should be implemented. As it is kinda hacky to just add HBO container + button to the 2D/3D/Script/Asset Store buttons.
But I thought it was quite appealing UX to have that button there

Reply:
> I wonder if there is a more robust way to do this, for example if mac or any OS changes their button layout this will fail.
> Is there a way to identify the container? If there is, we can do a loop and search for that specific container.

Potentially we could look for one of the other buttons. 3D,2D, Assets, or Script and then get their parent. The Buttons should have their respective names in their text property. So that's what we could check against

Reply:
Maybe but the top bar seem a nice place !

Godot contributors actually work on the same feature for the engine : https://github.com/godotengine/godot/pull/97257

And they put it on the top bar too, so If we want a more robust way to put the button on top maybe you can check the code they made to add the button on the top bar

Reply:
I wouldn't block on this PR tho. I think it would be great if we can land this to unblock mac users. We can always refine later.

Reply:
Thanks ! @nan0m I let you merge the PR if you are ok with that too :)

Reply:
@Meta-Ben I agree. I will merge for now, thank you!